python/lsst/ctrl/bps/wms_service.py

```diff
@@ -209,6 +209,15 @@ def cancel(self, wms_id, pass_thru=None):
         """
         raise NotImplementedError
 
+    def submission_checks(self):
+        """Checks to run at start if running WMS specific submission steps.
+
+        Any exception other than NotImplementedError will halt submission.
+        Submit directory may not yet exist when this is called.
+        """
+        _LOG.debug("submission_checks is not implemented.")
```

Comment:
Is there a reason behind putting this log message here instead of the try-except clause in `drivers.py` (lines 123-128)?

Reply:
I went back and forth about where to put the debug message.  Since it was a debug level message I ended up putting it here in the common code instead of where every code that calls it would have to repeat it.